packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -20,15 +20,23 @@ interface IElectionModule {
 
     function setSeatCount(uint seats) external;
 
-    function setEpochDuration(uint duration) external;
+    function setEpochDuration(uint64 duration) external;
 
     function setPeriodPercent(uint8 percent) external;
 
     function setNextSeatCount(uint seats) external;
 
-    function setNextEpochDuration(uint duration) external;
```

Comment:
Nit: Stick to a single naming pattern? If you look at the functions in this interface, some adhere to one pattern and some to the other.
Either `setNext<name>` or `setNextEpoch<name>`.

Reply:
It is consistent. We are setting/updating the NextEpochDuration. If we use here, i.e. setNextDuration, we can don't know which duration. Also this adhere with the naming on the specs. https://www.notion.so/GOV-v3-Spec-a045502acd93481182541441e4cf5ddd

Reply:
Gotcha!